content/layout/intrinsic-0_overview/index.md

```diff
@@ -0,0 +1,73 @@
+---
+page_type: guide
+title: Overview
+author: adamargyle
+description: A introduction article that sets up the tasks and goals of the guide
+tags:
+- Layout
+- Design
+- CSS
+web_lighthouse: N/A
+web_updated_on:
+web_published_on:
+wf_blink_components: Blink>Accessibility
+---
+
+Heard this term yet in the context of web, [intrinsic](https://adactio.com/journal/13671)? [Jen Simmons](https://shoptalkshow.com/episodes/328-jen-simmons-intrinsic-web-design/) **astutely coined it**. I love it. No, like **loooooove** it, and I must **share** with you **the joy**.
+
+<figure style="text-align:center; margin: 5rem 0;">
+  <img src="https://media3.giphy.com/media/26FLdmIp6wJr91JAI/giphy.gif?cid=3640f6095c9541ae7945334751d09c8b" alt="Patrick Star admiring with floating hearts">
+</figure>
+
+Intrinsic has a few synonyms or alternative implecations and meanings I want to share quickly too, since I think they can really help you shape a mental model of what we're going for:
+1. [Intrinsic web design](https://twitter.com/jensimmons/status/980980521848127488?lang=en)
+1. [Content centric/outward layout](http://bradfrost.com/blog/post/7-habits-of-highly-effective-media-queries/)
+1. [Resilient CSS](https://www.smashingmagazine.com/2017/03/resilient-web-design/)
+1. Courteous CSS: okay, this one's mine 🤓
+
+<br><br>
+
+# Tangible Learning
+Intrinsic definitions and articles get abstract, so instead of define it for you again, **I'd rather define it tangibly and visually** with you.
+
+Our overarching task throughout this guide is to **lay out the following design,** as a practical way of learning the ropes. Check it out. It might not look like much, but it's **packed with learning opportunities!** 💀🤘
+
+<figure style="text-align:center; margin: 5rem 0;">
+  <img src="home.png" alt="TenHundred store home page" class="screenshot">
+  <figcaption><a href="https://argyleink-webdev-intrinsic-layout-guide.glitch.me/">Interactive Demo</a></figcaption>
+</figure>
+
+## Prepare to
+- lean into content variability: aka **assume text/images are out of our control**
+- simulate some **chaos**: aka **simulate that lack of content control**
+- lean into **[css grid](https://css-tricks.com/snippets/css/complete-guide-grid/)**
+- **avoid [fixed units](https://www.w3.org/TR/css-sizing-3/)**
+- **avoid defining heights** (though I'll demo exceptions!)
+- **avoid writing @media queries**
+- pacify our `grid-template-areas` knee jerk reactions
```

Comment:
great request, will find and link.
i really like this request because it's important that some of my claims are backed with examples, and many of them are likely claimed and not supported at the moment